src/services/fields.ts

```diff
@@ -192,12 +232,7 @@ export function getFilterTypeFromLabelType(type: LabelType | null, key: string,
       if (key === LEVEL_VARIABLE_VALUE) {
         return VAR_LEVELS;
       }
-      return VAR_FIELDS;
-    }
-    default: {
-      const err = new Error(`Invalid label type for ${key}`);
-      logger.error(err, { value, msg: `Invalid label type for ${key}` });
-      throw err;
+      return VAR_METADATA;
     }
   }
```

Comment:
LabelType can't be null anymore, and we already cover all the cases. I can add the default and throw an error back in, in case someone has a messed up dataframe that includes an unknown LabelType (not `I`, `S` or `P`)?

Reply:
Eh, doesn't hurt to fail loud, I added a default case that throws an error.